.gitignore

```diff
@@ -0,0 +1 @@
+docker-compose.override.yml
```

Comment:
Is this intentional? `docker-compose` isn't in this repo  : ]

Reply:
I removed the docker bits from this PR and will move it to a different one.